pyproject.toml

```diff
@@ -38,11 +41,12 @@ build = "0.10.0"
 certifi = "2022.12.7"
 charset-normalizer = "3.1.0"
 click = "8.1.3"
+django = "^4.2"
 docutils = "0.19"
 fastapi = "0.95.0"
 h11 = "0.14.0"
 idna = "3.4"
-importlib-metadata = "6.1.0"
+importlib-metadata = "~6.0.0"
```

Comment:
Downgraded because of opentelemetry (at least `opentelemtry-api`) 
https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/pyproject.toml#L32

Reply:
it might be beneficial to leave the reasoning behind the pin here